### DISCUSSION TITLE
GIT BACKUP

### DISCUSSION THREAD

**Discussion by Jonweb08** [Guilouz/Creality-Helper-Script-Wiki#537] [Issues]
I'm newbye, I installed Git backup on my K1 MAX trough your script, i follow the procedure  but I'm not able to see the file in github. I decide to remove  it and re install, but it  seems not possible or better the message said termineted, but on help script menu remained. I try also to reinstall it but they give me a error and teh script sugegest to delete the file manually.  Probably i made the mistake and i delete the folder .git.  Now another message apperead. Can you help me please? 
Regards
![tempsnip](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/81989253/60b2b315-78fa-4dda-a8a5-6b748ece7d96)

**Comment by Guilouz** [comment]
Fixed with last release.

**Reply by hojen1234** [reply]
Root my K1max with your latest scripts.   GIT backup seemed to install properly first time, but didn't appear to be working, so I uninstalled it and tried to reinstall it with your scripts.   I consistently get the following error on attempted re-installation:
![Clipboard01](https://github.com/user-attachments/assets/6e5d0a01-8847-4650-ab0f-e570a51b581d)

This happens despite the repository being empty, and verifed to have no commits.    I have deleted the old repository, created a new empty repository and renewed my token, all with no success.   I have asked others to verify that my Github account and its single repository are in working order, ...     Do you have an suggestions for me ?    I am tempted to do the Git Backup installation manually, from the instructions on its web page, but would rather remain within the boundaries of your excellent installation scripts.

**Reply by hojen1234** [reply]
this also happens if I create new branch different from "main":
![Clipboard01](https://github.com/user-attachments/assets/99bb4557-b394-4234-96a7-756cfbc2be7a)
